packages/web-components/fast-components/src/slider/slider.styles.ts

```diff
@@ -16,6 +17,18 @@ import {
     neutralOutlineRestBehavior,
 } from "../styles/index";
 
+const ltr = css`
+    :host(.vertical) .track {
+        margin-left: calc((var(--design-unit) + calc(var(--density) + 2)) * 1px);
```

Comment:
Could this be a CSS Logical property? Is it `margin-inline-start` or `margin-inline-end`?

Reply:
The answer is yes, I was just testing you and you passed! Updating now